dask/blockwise.py

```diff
@@ -196,7 +196,12 @@ def __dask_distributed_pack__(
             required_indices = tuple(self.mapping.keys())
 
         return {
-            "mapping": {k: to_serialize(self.mapping[k]) for k in required_indices},
+            "mapping": {
+                k: stringify(self.mapping[k])
```

Comment:
I don't really have a strong grasp on why we are stringifying keys on the scheduler in the first place, but in a world where we are doing that, this seems fine.

Reply:
>I don't really have a strong grasp on why we are stringifying keys on the scheduler in the first place

My understanding is that the reason is some mixture of msgpack list/tuple handling and general performance. Thereofre, if we continue down the "pickle on scheduler" route, stringification certainly seems like something we should reconsider :)